inst/tutorials/r_and_rstudio_basic/r_and_rstudio_basic.Rmd

```diff
@@ -85,12 +80,23 @@ Notice that the window has three "panes":
 To change the look of RStudio, you can go to Tools &rarr; Global Options &rarr; Appearance and select colours, font size, etc. If you plan on working for longer periods of time, we suggest choosing a dark background colour which is less hard on your computer battery and your eyes.
 You can also change the sizes of the panes by dragging the dividers or clicking on the expand and compress icons at the top right corner of each pane.
 
+### Check Your Understanding
+
+When trying to run a command, you see this error:
+
+![](/images/objecterror.png){width=100%}
+
 ```{r Tour, echo=FALSE}
 quiz(
-  question("Which pane enables you to manage R packages?",
+  question("Where is the command typed?",
+    answer("The console", correct=TRUE),
+    answer("Lower right pane"),
+    answer("Upper right pane")),
+  
+  question("Where can you go to try to locate 'object'?",
     answer("The console"),
-    answer("Lower right pane", correct=TRUE),
-    answer("Upper right pane"))
+    answer("Lower right pane"),
+    answer("Upper right pane", correct=TRUE))
```

Comment:
Yes, this is a neat idea. I like this style of questioning--introducing students to the types of common errors they might encounter is prudent. Well done.